### PR TITLE
[TemplateFlip]  -> [HTML & CSS Templates]

### PR DESCRIPTION
**Pull request by cetinogut**
Hi, 
I found this resource useful for   all community members. Regards

https://templateflip.com/templates/

### CODE REVIEW

**Comment by cetinogut** [comment]
Hi Alaev,
I found this resource also useful for all community members. Regards

https://www.bootstrapdash.com/product-category/free-bootstrap-admin-templates/

File: readme.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```
Comment (StefanSelfTaught) [inline readme.md:6]: Don't think we need this link here.

**Comment by StefanSelfTaught** [comment]
Why did you removed the templateflip link?

**Comment by cetinogut** [comment]
Hi Stefan, sorry for the confusion. In fact, I had two additions. No removal.. First one is Templateflip, the second one is Bootstrapdash.

**Comment by StefanSelfTaught** [comment]
Ok, please add the other resource and remove table of content link.

**Comment by StefanSelfTaught** [comment]
@cetinogut

**Comment by Alaev** [comment]
Closing.

Not 100% free.

**Pull request closed without merging**